packages/cli/fragments/fragment-preact/src/app.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "preact/hooks";
+import preactLogo from "./assets/preact.svg";
+import "./app.css";
+
+export function App() {
+  const [count, setCount] = useState(0);
+
```

Comment:
Instead of a normal counter example, there should be an example here for using tauri's `invoke` api, you can check the react fragment to see how it is done.